src/main/java/seedu/duke/tasks/TaskList.java

```diff
@@ -0,0 +1,9 @@
+package seedu.duke.tasks;
```

Comment:
Not sure if this class was really needed.

Reply:
It was just to fulfil requirements 🙊